examples/src/simple.rs

```diff
@@ -386,8 +386,13 @@ fn interpret_input(input: &str, sys: &mut System) -> bool {
             );
             writeln!(
                 &mut io::stdout(),
-                "System version:   {}",
-                sys.get_version().unwrap_or_else(|| "<unknown>".to_owned())
+                "System kernel version:   {}",
+                sys.get_kernel_version().unwrap_or_else(|| "<unknown>".to_owned())
+            );
+            writeln!(
```

Comment:
Just realized that doing all of it in one `writeln!` would be much better. Also, please align the `{}`. :p

Reply:
fixed in 1aff6db